### PR TITLE
runqslower: Place generated skeleton in .output directory

### PR DESCRIPTION
**Pull request by danobi**
In line with how libbpf-bootstrap and libbpf-tools does it.

### CODE REVIEW

File: examples/runqslower/src/main.rs [hunk 1]
```diff
@@ -8,6 +8,7 @@ use libbpf_rs::PerfBufferBuilder;
 use plain::Plain;
 use structopt::StructOpt;
 
+#[path = "bpf/.output/mod.rs"]
```
Comment (anakryiko) [inline examples/runqslower/src/main.rs:11]: does it have to be called generic "mod.rs"? What will happen if application is using two different skeletons? Ideally each skeleton should be called with its name, something like `runqslower.skel.rs` and `some_other.skel.rs`

Comment (danobi) [reply examples/runqslower/src/main.rs:11]: That is controlled in the build script: https://github.com/libbpf/libbpf-rs/blob/699e2a7e6b3e49e212902ce95314f850e2ffdf4e/examples/runqslower/build.rs#L16-L17

If there's multiple skeletons, the user is free to name each one whatever they want. That's the nice thing about the `#[path=..]` attribute

Comment (anakryiko) [reply examples/runqslower/src/main.rs:11]: cool, can we please use `runqslower.skel.rs` to set a good example?

Comment (danobi) [reply examples/runqslower/src/main.rs]: Done

File: examples/runqslower/src/main.rs [hunk 2]
```diff
@@ -8,6 +8,7 @@ use libbpf_rs::PerfBufferBuilder;
 use plain::Plain;
 use structopt::StructOpt;
 
+#[path = "bpf/.output/runqslower.skel.rs"]
 mod bpf;
```
Comment (anakryiko) [inline examples/runqslower/src/main.rs:12]: hm.. I hope user can also control`mod bpf` name, but I'm obnoxious enough already, so let's land this for now.

**Review by anakryiko — APPROVED** [review]

File: examples/runqslower/src/main.rs (hunk 2 above)
Comment (willfindlay) [reply examples/runqslower/src/main.rs:12]: They should be able to. Here the `#[path]` decorator here is overloading the path for `mod bpf`, but in principle, you could name the generated skeleton whatever you like and the module name would correspond.

**Comment by danobi** [comment]
Updated to use `runqslower` instead of `bpf` for module name

**Pull request merged**